cuqi/distribution/_truncated_normal.py

```diff
@@ -0,0 +1,48 @@
+import numpy as np
+from scipy.special import erf
+from cuqi.distribution import Distribution
+
+class TruncatedNormal(Distribution):
+    """
+    Truncated Normal probability distribution. Generates instance of cuqi.distribution.TruncatedNormal.
+    
+    The variables of this distribution are iid.
+
+    
+    Parameters
+    ------------
+    mean: mean of distribution
+    std: standard deviation
+    a: lower bound of the distribution
+    b: upper bound of the distribution
```

Comment:
I am not sure what is the best name for the upper and lower bounds, I suggest, however, making them consistent with the uniform distribution. In the uniform distribution, we have 
```
        low : float or array_like of floats
            Lower bound(s) of the uniform distribution.
        high : float or array_like of floats 
            Upper bound(s) of the uniform distribution.
```
 It could be we also use `low` and `high` here, or we change the names in both distributions to `lower` and `upper` or something, what do you think?

Reply:
Sounds good to me.

Reply:
👍